src/components/navbar/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+import { TitleContent, HeaderContent, Title, SubTitle, Nav } from "./Styles";
+
+export default class NavBar extends Component {
+  //add state for controlling mobile nav
+  render() {
+    return (
+      <header>
+        <HeaderContent>
+          <TitleContent>
+            <Title>{title.name}</Title>
+            <hr />
+            <SubTitle>{title.subTitle}</SubTitle>
+          </TitleContent>
+          <Nav>
+            {links.map((link, index) => (
+              <li key={index}>
+                <a href={link.path}>{link.name}</a>
+              </li>
+            ))}
+          </Nav>
+        </HeaderContent>
+      </header>
+    );
+  }
+}
+
+const title = { name: "DRY CREEK INDUSTRIES", subTitle: "General Contracting" };
```

Comment:
IMO values you are initializing here should be at the top of the file, before any references.

Reply:
If you want to make this component more reusable you should pass these in as props.

Reply:
I can move them to the top, makes more sense to have it up there.

Reply:
I was going to pass the links array down as props for my mobile nav. I guess it doesn't make sense why I made the title and subtitle as an object.